task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs"
   name: deprecated-image-check
 spec:
+  description: >-
+    Check the images are not deprecated
```

Comment:
Does this check if the image being scanned is deprecated?

Or, does it check if the base image that the image is built on top of is deprecated?

Reply:
i'll make a copy change suggestion after you get your answer, @ralphbean

Reply:
It accepts an image list as a parameter and checks if they are not deprecated. In the pipeline, this image list are generated by buildah task

Reply:
Updated the `description` to `Checks the base images are not deprecated`